src/main/java/com/jcabi/github/RtCommitsComparison.java

```diff
@@ -94,4 +104,75 @@ public JsonObject json() throws IOException {
         return new RtJson(this.request).fetch();
     }
 
+    /**
+     * Iterator that yields FileChange objects converted
+     * from JSON objects in a JSON list.
+     */
+    @EqualsAndHashCode(of = { "iterator" })
+    @Loggable(Loggable.DEBUG)
+    @ToString
+    private static final class FileChangesIterator
+        implements Iterator<FileChange> {
+        /**
+         * Encapsulated iterator of file change JSON objects.
+         */
+        private final transient Iterator<JsonObject> iterator;
+
+        /**
+         * Ctor.
+         * @param iter Iterator of file change JSON objects
+         */
+        FileChangesIterator(
+            @NotNull(message = "files can't be NULL")
+            final Iterator<JsonObject> iter
+        ) {
+            this.iterator = iter;
+        }
+
+        @Override
+        public FileChange next() {
+            return new RtFileChange(this.iterator.next());
```

Comment:
@cvrebert You should be able to use the [SuppressFBWarnings](http://findbugs.sourceforge.net/api/edu/umd/cs/findbugs/annotations/SuppressFBWarnings.html) annotation: `@SuppressFBWarnings("IT_NO_SUCH_ELEMENT")`